src/layouts/HeaderWrapper/HeaderWrapper.jsx

```diff
@@ -0,0 +1,12 @@
+import Header from '../../components/Header/Header';
+import Container from '../../components/Container/Container';
+
+function HeaderWrapper({ themeMode, toggleThemeMode }) {
+  return (
+    <Container>
+      <Header themeMode={themeMode} toggleThemeMode={toggleThemeMode} />
+    </Container>
+  );
+}
+
+export default HeaderWrapper;
```

Comment:
You can use _Container_ component in _Header_ component if you need to use container. And then you don't need this wrapper.